triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractTriggerTest.scala

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.daml.lf.engine.trigger.test
```

Comment:
Why not pull this out into a separate `scala_library` for testing utils and have the actual tests stay in a `scala_test_suite`? If not for parallelization, it also just seems cleaner to separate those concerns.

Reply:
I’ve pulled it into a `test-utils` library now. Parallelizing it made the tests run slower when run in parallel (presumably JVM overhead) so I’ve kept it in a single test suite for now.

Reply:
Thanks!